### PR TITLE
Reusing the code of semantic segmentation for node example

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@huningxin, PTAL, thanks!

**Comment by Honry** [comment]
@mingmingtasd, since @huningxin is OOO, could you help review this PR?

File: object_detection/index.html [hunk 1]
```diff
@@ -140,8 +140,8 @@ <h2 class="text-uppercase text-warning">No model selected</h2>
     integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
     crossorigin="anonymous"></script>
   <script src="https://webmachinelearning.github.io/webnn-polyfill/dist/webnn-polyfill.js"></script>
-  <script id="tf" src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@latest/dist/tf.min.js" 
-    integrity="sha256-B72Do2A4yC67Y3aXueG3wQXWYjtr5RlI1D2jYDJJYsw=" 
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@latest/dist/tf.min.js"
```
Comment (huningxin) [inline object_detection/index.html:143]: should we align the tf.js version with webnn-polyfill?

File: semantic_segmentation/index.html [hunk 2]
```diff
@@ -184,6 +184,9 @@
     crossorigin="anonymous"></script>
   <script src="https://cdn.jsdelivr.net/npm/@jaames/iro@5"></script>
   <script src="https://webmachinelearning.github.io/webnn-polyfill/dist/webnn-polyfill.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@latest/dist/tf.min.js"
```
Comment (huningxin) [inline semantic_segmentation/index.html:187]: ditto

File: object_detection/index.html (hunk 1 above)
Comment (Honry) [reply object_detection/index.html]: I think so, thanks, done.

**Review by huningxin — APPROVED** [review]
LGTM, thanks @Honry !

**Pull request merged**